cypress/integration/percentile-calculator-page.spec.ts

```diff
@@ -33,6 +33,24 @@ describe('Percentile Calculator Page', () => {
     })
   })
 
+  describe("Id's exists - for analytics", () => {
+    it('The reset button exists', () => {
+      cy.get('#reset-percentiles-button').should('exist')
```

Comment:
`.should('exist')` is actually redundant, because it will fail if can't find the element

Reply:
Oh - so just cy.get("#thing") is enough? Ok (doesn't read as clearly - but ok).